framework/helpers/BaseArrayFilterHelper.php

```diff
@@ -0,0 +1,249 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\helpers;
+
+use yii\base\InvalidParamException;
+
+/**
+ * BaseArrayFilterHelper provides concrete implementation for [[ArrayFilterHelper]].
+ *
+ * Do not use BaseArrayFilterHelper. Use [[ArrayFilterHelper]] instead.
+ *
+ * @author Daniil Razorenov <daniltmb@mail.ru>
+ */
+class BaseArrayFilterHelper
```

Comment:
This class contains a single public method, thus this is not a helper. It must be implemented not as a class of static method.